install_config/aggregate_logging.adoc

```diff
@@ -937,6 +962,65 @@ existing `journald` log driver configuration. It is recommended to use the
 Logging Driver] for instructions to change your existing log driver
 configuration to `json-file`.
 
+[[fluentd-file]]
+*Configuring Fluentd to Send Logs to a File*
+
+Fluentd writes logs to a file you specify with the `LOGGING_FILE_PATH` environment variable, by default `/var/log/fluentd/fluentd.log`.  
```

Comment:
Fluentd writes its logs to console if LOGGING_FILE_PATH=console, it's not a file, so I think "Fluentd writes logs to a file you specify with the `LOGGING_FILE_PATH` environment variable" is not accurate.